src/js/modules/popupRegistration.js

```diff
@@ -54,13 +51,22 @@ function gratitudeOpen() {
 	registerModalGratitude.classList.add("open");
```

Comment:
you use `gratitudeOpen` only in `submitHandler`, I think you can just add `registerModalGratitude.classList.add("open");` to `submitHandler` and remove `gratitudeOpen`

by the way, it is a bad name for the function, it must be something like `openGratitude`

Reply:
Done,